tests/PHPStan/Analyser/data/pdo-quote.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace PDOQuote;
+
+use function PHPStan\Testing\assertType;
+
+class Foo
+{
+
+	public function doFoo()
+	{
+		$database = new \PDO('dsn');
+		$quote_string = $database->quote('string');
+		assertType('(string|false)', $quote_string);
```

Comment:
per function map this should be

```suggestion
		assertType('string|false', $quote_string);
```